### PR TITLE
waf: build sitl on windows

### PR DESCRIPTION
**Pull request by lucasdemarchi**
Tested on cygwin - only things that need to be installed are gcc, g++ and python.

### CODE REVIEW

File: Tools/ardupilotwaf/boards.py [hunk 1]
```diff
@@ -177,6 +177,11 @@ def configure_env(self, cfg, env):
             'SITL',
         ]
 
+	if sys.platform == 'cygwin':
```
Comment (OXINARF) [inline Tools/ardupilotwaf/boards.py]: The if seems misaligned? It also seems to be using a tab instead of spaces.

Comment (lucasdemarchi) [inline Tools/ardupilotwaf/boards.py]: yeah... editing on windows is not nice. Thanks.

**Comment by guludo** [comment]
Besides the unintentional use of TAB instead of spaces, this looks good to me. Thanks.

**Comment by lucasdemarchi** [comment]
I fixed the tab and applied.

**Pull request closed without merging**